client/src/css/app.sass

```diff
@@ -72,6 +72,13 @@ a[target="_blank"]::after
         .q-field__control
             height: auto
 
+th.text-right, th.text-left, th.text-center, .q-table tbody td, .q-table__bottom-item,
```

Comment:
Submission Export
![Screenshot of Submission Export showing 14px font size](https://github.com/MESH-Research/Pilcrow/assets/2668987/131898e2-b8d8-47d1-9896-799ba26d3887)

Reply:
The paragraphs now appear as 16px, although I would have put the `.text-body1` class on the `main` element to simplify the application of font size so it doesn't have to be applied per child element.